src/app/_variables.scss

```diff
@@ -147,3 +141,51 @@ produces
 @function dim-rgb-values-to-rgba($values, $alpha: 1) {
   @return #{'rgba(' + $values + ', ' + $alpha + ')'};
 }
+
+// Theme variables
+:root {
+  // Font
+  --theme-font-primary: 'Open Sans';
+
+  // Base colours
+  --theme-accent-primary: #e8a534;
+  --theme-accent-secondary: #68a0b7;
+
+  --theme-fill-primary: #000; // Base popups, drawers, dropdowns
+  --theme-fill-secondary: #161616; // Layered dropdowns inc search, item actions
+
+  // Fills to tint panels and sub-frames e.g. masterwork/catalyst progress
+  --theme-fill-lighten: rgba(255, 255, 255, 0.15);
+  --theme-fill-darken: rgba(0, 0, 0, 0.35);
+
+  --theme-button: rgba(255, 255, 255, 0.2);
```

Comment:
Recommend keeping as-is. 

The currently transparent tinted buttons work very well since they are used against loads of slightly different background colours.

Reply:
That's fair. We can keep it as is.